eventtracking/backends/logger.py

```diff
@@ -45,7 +45,7 @@ def send(self, event):
 class DateTimeJSONEncoder(json.JSONEncoder):
     """JSON encoder aware of datetime.datetime and datetime.date objects"""
 
-    def default(self, obj):  # pylint: disable=method-hidden
+    def default(self, obj):  # lint-amnesty, pylint: disable=arguments-differ, method-hidden
```

Comment:
Aren't these `lint-amnesty` suppressions supposed to be added in one fell swoop using the automated method described here?:
https://github.com/edx/edx-lint#using-lint-amnesty